lib/tool-bar-view.coffee

```diff
@@ -46,29 +44,38 @@ module.exports = class ToolBarView extends View
       @updatePosition 'Left'
       atom.config.set 'tool-bar.position', 'Left'
 
-    atom.config.observe 'tool-bar.iconSize', (newValue) =>
+    @subscriptions.add atom.config.observe 'tool-bar.iconSize', (newValue) =>
       @updateSize newValue
 
-    atom.config.onDidChange 'tool-bar.position', ({newValue, oldValue}) =>
+    @subscriptions.add atom.config.onDidChange 'tool-bar.position', ({newValue, oldValue}) =>
       @show() if atom.config.get 'tool-bar.visible'
 
-    atom.config.onDidChange 'tool-bar.visible', ({newValue, oldValue}) =>
+    @subscriptions.add atom.config.onDidChange 'tool-bar.visible', ({newValue, oldValue}) =>
       if newValue then @show() else @hide()
 
     if atom.config.get 'tool-bar.visible'
       @show()
 
-    @.element.addEventListener 'scroll', @drawGutter
+    @on 'scroll', @drawGutter
```

Comment:
We want to move away from jQuery as much as possible (for the moment `space-pen` is dropped). I also don't see any reason to change this. Can you confirm this?